ci/setup.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+# setups test with required packages
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+KIND_VERSION=v0.7.0
+HELM_VERSION=v3.2.0
+
+echo "=== Install kubectl"
+sudo curl -L https://storage.googleapis.com/kubernetes-release/release/$(curl -s https://storage.googleapis.com/kubernetes-release/release/stable.txt)/bin/linux/amd64/kubectl -o /usr/local/bin/kubectl && sudo chmod +x /usr/local/bin/kubectl
```

Comment:
We don't need installing kubectl. At least it's not being used for what I could see

Reply:
This is for future proofing, will use when doing `kind` cluster testing for e2e